src/post.pegjs

```diff
@@ -88,6 +90,22 @@ HTML_Comment
     type: "HTML_Comment",
     innerText: cs.join('')
   } }
+
+HTML_Tag_Void
+  = t:HTML_Tag_Open
+  & { return undefined !== {
+      'br': true,
+      'col': true,
+      'embed': true,
+      'hr': true,
+      'img': true,
+      'input': true
```

Comment:
it's not entirely clear in the spec but it is deduced but the rules

https://www.w3.org/TR/html5/syntax.html#parsing-main-inbody
then search for "area, br" and it should jump to the right paragraph

> Insert an HTML element for the token. Immediately pop the current node off the stack of open elements.

This means insert the DOM node and then stop processing `img` and friends. If we encounter an end tag for an image it means we get a parse error.